tf/vault.tf

```diff
@@ -0,0 +1,66 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "gratibot" {
+  name                       = "Gratibot"
```

Comment:
Should we rename this resource to include the environment? Otherwise I believe the `secret_uri` ends up being `https://gratibot.vault.azure.net/` which we wouldn't be able to differentiate from prod.